PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -517,46 +548,56 @@ struct HfTrackIndexSkimCreatorTagSelTracks {
         }
       }
       if (!hasGoodQuality) {
-        statusProng = 0;
         MY_DEBUG_MSG(isProtonFromLc, LOG(info) << "proton " << indexBach << " did not pass clusters cut");
-        if (debug) {
-          for (int iCandType = 0; iCandType < CandidateType::NCandidateTypes; iCandType++) {
-            // cutStatus[iCandType][2] = false;
```

Comment:
You can remove all occurrences of `cutStatus` in this function as it does not seem used anywhere. Maybe the `debug` switch can then be completely removed.

Reply:
Done! I removed `debug` from the strcut for the track selection, however I would prefer not to do it in the other structs not to mix too many things in this PR, which should be dedicated to the D* implementation, and eventually do it in another PR for code cleaning/optimisation.

Reply:
Thanks, yes, I meant removing it only in the concerned struct.